CONTRIBUTING.md

```diff
@@ -85,7 +85,7 @@ project's repository.
     git remote add upstream https://github.com/vmware-tanzu/antrea
     ```
 
-    Never push to upstream master
+    Never push to upstream remote
```

Comment:
remote or main?

Reply:
remote, I think the earlier version was inaccurate. The command is not branch-specific but applies to the remote.

Reply:
Got it. Thank you!